app/services/report_importer.rb

```diff
@@ -95,7 +83,11 @@ def import_log_messages
     end
   end
 
-  def inspect_report
+  def report_status
+    ConfigReportStatusCalculator.new(:counters => raw['status']).calculate
```

Comment:
Comment still relevant I think, should this method impl be moved and this raise an error?